tests/tensor_search/integ_tests/test_add_documents_structured.py

```diff
@@ -915,4 +915,26 @@ def test_long_double_numeric_values_edge_case(self):
                 returned_doc = tensor_search.get_document_by_id(
                     config=self.config, index_name=self.index_name_1, document_id=document_id, show_vectors=False
                 )
-                self.assertEqual(expected_doc, returned_doc)
\ No newline at end of file
+                self.assertEqual(expected_doc, returned_doc)
+
+    def test_add_documents_nonImageContentForAnImageField(self):
+        """Test to ensure a proper error is raised when non-image content is added to an image field"""
+        documents = [
+            {
+                "_id": "1",
```

Comment:
I think having 3 docs would make this more of an average case. I would probably also test having 1 doc with bad image URL, having 2 docs with bad image URLs, having multiple bad image URLs in one doc

Reply:
Done